action.yml

```diff
@@ -19,28 +19,28 @@ description: >
   Commits standard using Commisery
 
 branding:
-  icon: 'code'  
+  icon: 'code'
   color: 'white'
 
 inputs:
-  token: 
+  token:
     description: 'GitHub token used to access GitHub (eg. github.token)'
     required: true
 
   validate-pull-request:
     description: 'Validates the Pull Request title and body'
     required: false
-    default: true
+    default: "true"
 
   validate-pull-request-title-bump:
     description: 'Validates the version bump level of the Pull Request title against the highest bump level in its commits'
     required: false
-    default: true
+    default: "true"
 
   validate-commits:
     description: 'Validates the commits within the current context'
     required: false
-    default: true
+    default: "true"
```

Comment:
The quotes are superfluous and blame history is worth more than some whitespace imperfections.
Please just revert this file.

Reply:
I get an error notification in the editor as the schema expect a string here. I can revert it, but they're not completely superfluous.
docs: https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputsinput_iddefault